src/components/ArticlesList.jsx

```diff
@@ -1,29 +1,67 @@
-import React, { useEffect, useState } from "react";
 import axios from "axios";
-import ArticleCard from "./ArticleCard";
-import {useParams} from "react-router-dom"
+import React from "react";
+import { useState, useEffect } from "react";
+import ArticleCard  from "./ArticleCard";
 
-export default function ArticlesList () {
-    const [articlesList, setArticlesList] = useState([])
-    
-    
-    useEffect(() => {
-        axios
-          .get("https://be-nc-news-tnfa.onrender.com/api/articles")
-          .then(({ data }) => {
-            console.log(data.articles)
-            setArticlesList(data.articles);
-          });
-      }, []);
+export default function topicsForm() {
```

Comment:
This function should still be named `ArticlesList` because that is still what it is responsible for, and that is still the file name